autosklearn/estimators.py

```diff
@@ -665,13 +665,15 @@ def fit(self, X, y,
         # type of data is compatible with auto-sklearn. Legal target
         # types are: binary, multiclass, multilabel-indicator.
         target_type = type_of_target(y)
-        if target_type in ['multiclass-multioutput',
-                           'continuous',
-                           'continuous-multioutput',
-                           'unknown',
-                           ]:
-            raise ValueError("classification with data of type %s is"
-                             " not supported" % target_type)
+        supported_types = ['binary', 'multiclass', 'multilabel-indicator']
```

Comment:
In the example you gave a link to scikit-learn's explanation of task types. Could you please add it to the error message here, too?

Reply:
Understood, I will add this in the next push.